### ISSUE TITLE
longer loading time of qmca with X

### ISSUE THREAD

**Issue by ye-luo** [QMCPACK/qmcpack#178]
When I use ssh with X, the same qmca command takes much longer time to initialize.
Since I don't use the plotting features very often, I would like to have an option like emacs -nw to bypass all the modules requiring X.

**Comment by prckent** [comment]
Good suggestion.

**Comment by prckent** [comment]
@markdewing did this get merged in?

**Comment by markdewing** [comment]
No, my attempted fix did not get merged.   I don't remember why.  Maybe I never made PR from it.

**Comment by jtkrogel** [comment]
Unimportant
